runtime/vm/ClassInitialization.cpp

```diff
@@ -391,6 +391,14 @@ classInitStateMachine(J9VMThread *currentThread, J9Class *clazz, J9ClassInitStat
 					clazz = VM_VMHelpers::currentClass(clazz);
 				}
 				VM_ObjectMonitor::exitObjectMonitor(currentThread, initializationLock);
+				/*Initialize a Flattened Class Cache */
+				j9object_t classReferenceObject = vm->memoryManagerFunctions->J9AllocateObject(currentThread, clazz, J9_GC_ALLOCATE_OBJECT_TENURED | J9_GC_ALLOCATE_OBJECT_NON_INSTRUMENTABLE);
```

Comment:
Not expert on this, but at another spot where class object was allocated. hash flag was requested, too.

Reply:
Yes, I also tried with hash flag at the beginning. With the hash flag, it throws a segmentation fault and it does not seem to be allocating the memory at correct address if I trace back. It seems to be working after removing the flag, so I am not sure.

Reply:
Value Types do not have identity hash codes so the hashcode flag is not applicable for defaultValues